html/changelogs/seeindark_bugfix.yml

```diff
@@ -0,0 +1,6 @@
+author: Jeser
+
+delete-after: True
+
+changes: 
+  - bugfix: "Removed see_in_dark value changes that resulted in necromorphs not having dark vision."

```

Comment:
Very minor comment, i wont require this change but:

Generally when writing changelogs, remember that they're going to be read by users with no technical knowledge. In this case it should say something like "Fixed the bug that caused necromorph nightvision to not work in complete darkness"